packages/karma-esm/src/setup-es-dev-server.js

```diff
@@ -81,6 +114,7 @@ async function setupDevServer(karmaConfig, esmConfig, watch, babelConfig, karmaE
     typeof esmConfig.port === 'number' ? esmConfig.port : await portfinder.getPortPromise();
   const karmaHost = `${karmaConfig.protocol}//${karmaConfig.hostname}:${karmaConfig.port}`;
   const devServerHost = `${karmaConfig.protocol}//${karmaConfig.hostname}:${devServerPort}`;
+  const importMap = await loadImportMap(esmConfig.importMap);
```

Comment:
I think its better to check if there is an importMap property and conditionally load it, that makes it easier to see that its an optional property.

Reply:
Done